### DISCUSSION TITLE
What is the need for a new cloud foundry provider as we already have one from cf community?

### DISCUSSION THREAD

**Discussion by codeyogi911** [SAP/terraform-provider-cloudfoundry#123] [Q&A] [answered]
Exisiting [CF provider](https://registry.terraform.io/providers/cloudfoundry-community/cloudfoundry/latest) is already available. What does this one provide over and above that one?

**Comment by Dray56** [comment, marked as answer]
Hi @codeyogi911 ,
You can refer the motivation section in [here](https://community.sap.com/t5/technology-blogs-by-sap/terraform-cloud-foundry-provider-for-sap-btp/ba-p/13695076) for your query.
